source/_integrations/ukraine_alarm.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: Ukraine Alarm
+description: Instructions on how to integrate Ukraine Alarm within Home Assistant.
+ha_category:
+  - Other
+ha_release: 2022.6
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@PaulAnnekov'
+ha_domain: ukraine_alarm
+ha_platforms:
+  - binary_sensor
+ha_integration_type: integration
+---
+
+The Ukraine Alarm integration uses the [Ukraine Alarm](https://www.ukrainealarm.com/) web service as a source for air-raid sirens in Ukraine for your region. It's became very important since [Russian troops invasion](https://war.ukraine.ua/).
+
+For each region integration exposes 4 binary sensors:
+
+- Air
+- Unknown
+- Artillery
+- Urban Fights
```

Comment:
```suggestion
The Ukraine Alarm integration uses the [Ukraine Alarm](https://www.ukrainealarm.com/) web service to offer alerts. The integration will create 4 binary sensors for your selected region in Ukraine:

- Air
- Artillery
- Urban Fights
- Unknown
```

Reply:
> to offer alerts

Actually, the official term is "air-raid siren" or "civil defense siren" (https://en.wikipedia.org/wiki/Civil_defense_siren). "alerts" is too generic. I've merged your suggestion, but left the term. Is it ok?